README.md

```diff
@@ -11,38 +13,67 @@ A python package for music and audio analysis.
 [![codecov](https://codecov.io/gh/librosa/librosa/branch/main/graph/badge.svg?token=ULWnUHaIJC)](https://codecov.io/gh/librosa/librosa)
 [![Docs](https://github.com/librosa/librosa/actions/workflows/docs.yml/badge.svg)](https://github.com/librosa/librosa/actions/workflows/docs.yml)
 
-Documentation
--------------
+ ##  Table of Contents
+
+**[Documentation](#Documentation)**<br>
+**[Installation](#Installation)**<br>
+
+- **[Using PyPI](#using-pypi)** <br>
+- **[Using Anaconda](#using-anaconda)** <br>
+- **[Building From Source](#building-from-source)** <br>
+- **[Hints for Installation](#hints-for-the-installation)** <br>
+
+**[Discussion](#Discussion)**<br>
+**[Citing](#Citing)**
+
+
+## Documentation
+
+
 See https://librosa.org/doc/ for a complete reference manual and introductory tutorials.
 
 The [advanced example gallery](https://librosa.org/doc/latest/advanced.html) should give you a quick sense of the kinds
 of things that librosa can do.
 
-Installation
-------------
+## Installation
+
+
+### - Using PyPI
```

Comment:
It seems strange to have section headings start with a hyphen.  Do we need it?

Reply:
The reason I have added hyphen in the beginning was to make them look like a subsection under installation. I can change it if it looks odd

Reply:
Got it - I think it's better without hyphens here.